### DISCUSSION TITLE
2025 Academic Conference 추천 책

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#428] [General]
2024 academic conference가 이번 주 금요일 마지막 sprint만 남겨두고 있고
워크샵 일정도 계획되어 있습니다.

매년 진행한 내용이지만 미리 추천 책 목록을 지정해 두고
6월 28일 금요일 오후 워크샵 때 모인 인원이 내년에 읽을 책 세 권을 결정하는 걸로 해보도록 하겠습니다.

추천 책 목록 적는 방법은
이 글 스레드를 통해 내년에 진행했으면 하는 책 제목을 권 수와 상관 없이 올려주시면 됩니다.

매년 책 추천 글 스레드는 아래 링크를 참고하시면 됩니다.
또 어떤 책을 고를지 모를 경우에도 아래 링크에 매년 많은 분들이 많은 책을 선정해 주셨기 때문에
그 중에 골라 보는 것도 방법일 수 있습니다.

- https://github.com/ThinkAboutSoftware/AcademicConference/discussions/329
- https://github.com/ThinkAboutSoftware/AcademicConference/issues/196

**Comment by yeslee-v** [comment]
- [HTTP 완벽 가이드](https://www.aladin.co.kr/shop/wproduct.aspx?ItemId=49731592)

이 책은 전에 모임에서 읽었던 책인데 다시 한 번 읽어보고 싶어 추천합니다.

**Comment by fkdl0048** [comment]
- [CODE](https://www.yes24.com/Product/Goods/16667186)

저는 이 책 추천합니다!

**Comment by jongfeel** [comment]
태형님 추천 책
도메인 주도 설계란 무엇인가?

http://aladin.kr/p/vFpYb

**Comment by jongfeel** [comment]
제 추천 책
더 나은 프로그래머 되는 법

**http://aladin.kr/p/yqSGA**

**Comment by jongfeel** [comment]
업서님 추천 책
타입으로 견고하게 다형성으로 유연하게

http://aladin.kr/p/c4Zpt
